apps/condo/domains/property/schema/Property.js

```diff
@@ -283,7 +293,7 @@ const Property = new GQLListSchema('Property', {
             },
         },
     },
-    plugins: [uuided(), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
+    plugins: [uuided(), addressService('address', { address: addressFieldHooks }), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
```

Comment:
it's better to leave the 'address' field as default value

Reply:
Yes, already done.
<img width="939" alt="image" src="https://user-images.githubusercontent.com/25384290/203053476-e5d7b572-0c88-4503-93f6-770e269edfcb.png">

But I have to pass this parameter here because of 2nd parameter exists.